### PR TITLE
Add RBC enabled log.

### PR DESCRIPTION
**Pull request by YuanjunHuang**
This is for the RBC feature test case, providing information to
indicate if RBC is working.

Jira: https://jira01.devtools.intel.com/browse/OAM-65434
Tests: test on APL.
Signed-off-by: Yuanjun Huang <yuanjun.huang@intel.com>

### CODE REVIEW

**Comment by YuanjunHuang** [comment]
@js0701 PTAL.

File: os/platformcommondrmdefines.cpp [hunk 1]
```diff
@@ -79,6 +79,10 @@ int CreateFrameBuffer(
     ret = drmModeAddFB2WithModifiers(
         gpu_fd, iwidth, iheight, iframe_buffer_format, m_igem_handles,
         m_ipitches, m_ioffsets, modifiers, fb_id, DRM_MODE_FB_MODIFIERS);
+
+    if (ret == 0) {
```
Comment (js0701) [inline os/platformcommondrmdefines.cpp]: there are a few modificators that is not related with RBC as I know.
For example, Yf tile.
We need to check exactly the CCS modificator I think.

Comment (tpalli) [reply os/platformcommondrmdefines.cpp]: IIRC Y-tiling is a requirement for RBC though so it is required and RBC will fail without it. What's the current set of modifiers in use?

Comment (YuanjunHuang) [reply os/platformcommondrmdefines.cpp]: both I915_FORMAT_MOD_Y_TILED_CCS and I915_FORMAT_MOD_Yf_TILED_CCS modifier are possible. In current impl, when the plane initialize, property will be queried what modifier is supported and set it to be prefer modifier, with which later to allocate offscreen render target and frame buffer. If allocate fails, will add it into blacklist. so I think we dont have to worry about this.

**Comment by js0701** [comment]
How about I915_FORMAT_MOD_Yf_TILED it is modificator, and not related with RBC?

File: os/platformcommondrmdefines.cpp (hunk 1 above)
Comment (js0701) [reply os/platformcommondrmdefines.cpp]: I915_FORMAT_MOD_Yf_TILED how about this one

Comment (YuanjunHuang) [reply os/platformcommondrmdefines.cpp]: Only I915_FORMAT_MOD_Y_TILED_CCS and I915_FORMAT_MOD_Yf_TILED_CCS is applicable for RBC, for primary plane. I915_FORMAT_MOD_Yf_TILED is not taking account.

**Comment by YuanjunHuang** [comment]
@js0701 update

**Comment by js0701** [comment]
LGTM

**Comment by js0701** [comment]
Closed as merged
